controllers/post.ctrl.js

```diff
@@ -48,16 +49,24 @@ const getAllPosts = async (req, res) => {
 const updatePost = async (req, res) => {
     try {
         const { id } = req.params; 
-        const { content, post_img } = req.body;
+        const { content, postImg } = req.body;
         const userId = req.session.user.id;
 
         if (!id) return sendResponse(res, 400, false, "post_id는 필수입니다.");
-        if (!post_img) return sendResponse(res, 400, false, "게시물 이미지는 필수입니다.");
 
-        // 이미지 업로드
-        const imageUrl = await uploadBase64ImageToS3(post_img);
+        const existingPost = await Post.getPost(id);
 
-        const result = await Post.updatePost(id, content, imageUrl, userId);
+        const existingImg = existingPost.postImg;
+    
+        // 새로운 이미지 업로드
+        let imageUrl = existingImg;
+        if (postImg) {
+            imageUrl = await uploadBase64ImageToS3(postImg);
+        }
+
+        const post = new Post({ content, postImg: imageUrl }, userId);
+
+        const result = await post.updatePost(id);
         
         if (result) {
```

Comment:
```suggestion
        if (!result) {
          return sendResponse(res, 403, false, "수정 권한이 없거나 게시물이 존재하지 않습니다.");
```
와 같이 early return 방식을 사용하면 불 필요한 조건문 하나를 없앨 수 있어서
코드 가독성이 더 올라갈 것 같습니다~

Reply:
오 이해했습니다!